core/sail/shacl/src/test/resources/test-cases/hasValue/targetShapeAnd/invalid/case2/report.ttl

```diff
@@ -4,17 +4,31 @@
 @prefix sh: <http://www.w3.org/ns/shacl#> .
 @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix rsx: <http://rdf4j.org/shacl-extensions#> .
 
 [] a sh:ValidationReport;
-  sh:conforms false;
   <http://rdf4j.org/schema/rdf4j#truncated> false;
+  sh:conforms false;
```

Comment:
What's the reason all these test case files change with almost every PR related to SHACL? I assume there's good reason but I find it a bit disconcerting to have a PR that is supposed to be a single bug fix touch 123 files, and (apparently) adjust test cases. Was the test wrong?

Reply:
Tests were wrong yes..

Reply:
Essentially:

```
rsx:targetShape [
	        sh:property [
        		sh:path ex:info ;
        		sh:and ([ sh:hasValue "blue" ; ]) ;
        	]
	];
```

Matches `ex:a` and `"red"` in `ex:a rdfs:label "red"` since neither of those nodes have an `ex:info` predicate. Previously I had only managed to match `ex:a`, even though I had intended to also match `"red"`.